addon/helpers/object.js

```diff
@@ -0,0 +1,43 @@
+import { MicroState } from 'ember-microstates';
+import assign from '../utils/assign';
+import { reduceObject } from '../utils/object-utils';
+
+export default MicroState.extend({
+
+  prototypeFor(value) {
+    return assign({}, value);
+  },
+
+  handlebarsValueFor(object, value) {
```

Comment:
I think this needs some documentation as to _what_ we're doing here. A lot of the code in the repo is self-evident when you look at it, but coming to this green and seeing it without an explanation, I'd be like wat?
```js
/**
 * Make all of the object properties explicitly enumerable in the template
 * 
 * ...explanation of why.
 */
```